pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs

```diff
@@ -137,6 +137,11 @@ public ImportTransitionListColumnSelectDlg(MassListImporter importer, SrmDocumen
             previousIndices = new int[Importer.RowReader.Lines[0].ParseDsvFields(Importer.Separator).Length + 1];
             InitializeComponent();
 
+            if (assayLibrary) // Dialog title should be "Import Assay Library: Identify Columns" instead of "Transition List: Identify Columns"
+            {
+                Text = Resources.ImportTransitionListColumnSelectDlg_Import_Assay_Library___Identify_Columns;
```

Comment:
Do we do this elsewhere? Are there any other forms like the error list form that still refer to a Transition List and not an Assay Library? We added File > Import > Assay Library fairly late in the game as a way of making more helpful decisions by default for a user that knows they have an "Assay Library" which is an extended form of a transition list with iRT values and relative ion intensities.

Reply:
I'll check.

Reply:
Well, there's this:
![image](https://user-images.githubusercontent.com/1200761/148287460-6d5ec587-22ba-422a-a8da-48b1ef683567.png)

and very briefly
![image](https://user-images.githubusercontent.com/1200761/148287740-6014b5f4-c7cb-4fc0-804d-7a5058b07018.png)

but I'm not really sure there's much to be gained by tweaking the language there.